src/common/src/array/data_chunk.rs

```diff
@@ -443,6 +444,45 @@ impl DataChunk {
             ..self
         }
     }
+
+    /// Serialize each rows into value encoding bytes.
+    ///
```

Comment:
How about adding some docs to show that:
- the len of return value is `cap` instead of `card`
- the invisible rows are filled empty.